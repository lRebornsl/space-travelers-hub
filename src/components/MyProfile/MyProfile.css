#reserved-container {
  display: grid;
  grid-template-columns: 1fr 1fr;
  gap: 0 3rem;
  margin-top: 1rem;
}

#reserved-container h2 {
  font-weight: normal;
  color: #484747;
  padding-bottom: 1rem;
}

#reserved-container li {
  color: #484747;
  padding: 2rem 2rem;
  border-bottom: 0.1rem solid #a4a4a4;
}

.profiles-container {
  border: 0.1rem solid #a4a4a4;
  border-radius: 3px;
}