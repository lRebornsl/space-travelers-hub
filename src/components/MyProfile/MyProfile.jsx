import ReservedRocketsList from '../Rockets/ReservedRocketsList';
import './MyProfile.css';

const MyProfile = () => (
  <div>
    <div id="reserved-container" className="container">
      <div>My Missions</div>
      <ReservedRocketsList />
    </div>
  </div>
)

export default MyProfile;