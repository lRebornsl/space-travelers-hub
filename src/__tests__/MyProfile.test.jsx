import { render, waitFor } from '@testing-library/react';
import { beforeEach, describe, it, expect } from 'vitest';
import { Provider } from 'react-redux';
import store from '../redux/Store';
import MyProfile from '../components/MyProfile/MyProfile';

describe('My Profile', () => {
  beforeEach(() => {
    render(
      <Provider store={store}>
        <MyProfile />
      </Provider>
    );
  });

  it('My Profile renders correctly', async () => {
    await waitFor(() => expect(MyProfile).toMatchSnapshot());
  });
});