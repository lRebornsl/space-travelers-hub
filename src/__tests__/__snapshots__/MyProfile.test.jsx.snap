// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`My Profile > My Profile renders correctly 1`] = `[Function]`;
